import-scripts/dmp-import-vars-functions.sh

```diff
@@ -319,7 +319,7 @@ function remove_raw_clinical_timeline_data_files {
     done
     # remove raw timeline files except cbio format timeline file
     for f in $STUDY_DIRECTORY/data_timeline*; do
-        if [ $f != *"data_timeline.txt"* ] ; then
+        if [[ $f != *"data_timeline.txt"* ]] ; then
```

Comment:
I don't understand this line ... the fix seems fine, but I don't get the string comparison to a de-globbed (wildcard expanded) list of filenames. But that is a pre-existing bug (in my opinion)